README.md

```diff
@@ -118,13 +118,49 @@ import project from '@dojo/web-editor/project';
 import Runner from '@dojo/web-editor/Runner';
 
 (async () => {
-	await project.load('some.project.json');
-	const runner = new Runner(document.getElementById('runner'));
-	await runner.run();
-	console.log('Ran!');
+    await project.load('some.project.json');
+    const runner = new Runner(document.getElementById('runner'));
+    await runner.run();
+    console.log('Ran!');
 })();
 ```
 
+### routing
+
+This module wraps `@dojo/routing` in a way that works well for the `web-editor`.
+
+The `web-editor` allows integration with GitHub's Gists, allowing storage of the `project.json` in a Gist which can then be used to load
+the project into the editor.
+
+#### setPath()
+
+Set the path on the current page.  Intended to be used, when a project is loaded to update the path to reflect the Gist's ID.
```

Comment:
"Intended to be used, when a project is loaded to update the path to reflect the Gist's ID." -> When a project is loaded, this is used to update the path to reflect the Gist's ID.